tests/unit/test_units.py

```diff
@@ -1,11 +1,87 @@
+import numpy as np
+
+
 from openscm.units import unit_registry
 from openscm.units import DimensionalityError, UndefinedUnitError, UnitConverter
 import pytest
 
 
 def test_unit_registry():
-    tq = unit_registry("CO2")
-    assert tq.to("C").magnitude == 12. / 44.
+    CO2 = unit_registry("CO2")
+    assert CO2.to("C").magnitude == 12.0 / 44.0
+
+
+def test_alias():
+    CO2 = unit_registry("carbon_dioxide")
+    assert CO2.to("C").magnitude == 12.0 / 44.0
+
+
+def test_base_unit():
+    # I don't know why this is this way, anyway
+    with pytest.raises(UndefinedUnitError):
+        unit_registry("carbon")
+
+
+def test_nitrogen():
+    N = unit_registry("N")
+    assert N.to("N2ON").magnitude == 28 / 14
+
+
+def test_nox():
+    # do we want to allow this?
```

Comment:
@znicholls what do you mean?

Reply:
If we let people convert NOx to N, then they can also convert NOx to N2O. Converting NOx to N2O doesn't make any sense and having that behaviour be allowed without at least a warning seems dangerous. Maybe it's something we can wrap in our converter class?

Reply:
Hmm, we could have warnings in the converter class, but that would be lot's of hacking, I am afraid (at least if we want a warning also for complex combinations of the units, e.g. with `kgN * m^2 / s` or so)

Reply:
Either way, are those TODO comments or meant for the user?

Reply:
They're comments to prompt this discussion haha

Thinking about it more, I think the cleanest solution would be to have contexts for NOx <--> N and CH4 <--> C conversions. That way they're still possible, you just have to be sure you want to do them and hence are less likely to accidentally get things wrong.

Reply:
cool ;)